docs_app/content/6-to-7-change-summary.md

```diff
@@ -163,7 +163,11 @@ This document contains a detailed list of changes between RxJS 6.x and RxJS 7.x,
 
 ### firstValueFrom
 
-- A better, more tree-shakable replacement for `toPromise()` (which is now deprecated). This function allows the user to convert any `Observable` in to a `Promise` that will resolve when the source observable emits its firsr value. If the source observable closes without emitting a value, the returned promise will reject with an `EmptyError`, or it will resolve with a configured `defaultValue`.
+- A better, more tree-shakable replacement for `toPromise()` (which is now deprecated). This function allows the user to convert any `Observable` in to a `Promise` that will resolve when the source observable emits its first value. If the source observable closes without emitting a value, the returned promise will reject with an `EmptyError`, or it will resolve with a configured `defaultValue`.
```

Comment:
> I should have reviewed it and included links to the markdown document in #6259.

@cartant Did you mean to add it to this file? Maybe like this:

```
(which is [now deprecated](/deprecations/to-promise))
```

Reply:
I'd add `Details: https://rxjs.dev/deprecations/to-promise` to the `toPromise` deprecations here:

https://github.com/ReactiveX/rxjs/blob/02a90983b1c0965140d49c2b0f999919db16545c/src/internal/Observable.ts#L486-L510

Kinda like this:

https://github.com/ReactiveX/rxjs/blob/02a90983b1c0965140d49c2b0f999919db16545c/src/internal/operators/publish.ts#L12-L18

Reply:
Or like this: https://github.com/ReactiveX/rxjs/pull/5949/files#diff-804122836d26336b7ccd318e42f664919dc07bf973ef16978e0c47753ff113abR510 😁

I'll do it, thanks for suggestion.